Added a few settings

Jazzelhawk opened the pull request:
Also added a DScrollPanel to the settings window
#344

davidsonbr commented:
Code style in editor.html

Edit: Not sure if html code style applies to the general code style.

Jazzelhawk committed:
Added word wrap and line folding keybind settings

Also added a DScrollPanel to the settings window

Jazzelhawk committed:
Added font size setting

Jazzelhawk commented:
Fixed the code style even though the editor.html doesn't really have a code style at the moment. It's kinda a mess. I fixed the merge conflicts and also added the font size setting.

Resolves #365
